### ISSUE TITLE
Multiline text

### ISSUE THREAD

**Issue by basdelfos** [gcanti/tcomb-form-native#9]
Using a multiline textinput generates an error message. The field is visible as a single line text input and is not editable.

Error shown in Xcode debug output:
`[RCTLog][tid:0x7fe59435fbc0][RCTUIManager.m:710]>No manager class found for view with module name "RCTTextView"`

Model:
```javascript
var Visit = t.struct({
  date: t.Dat,
  name: t.Str,
  email: t.Str,
  company: t.maybe(t.Str),
  comment: t.maybe(t.Str),
});
```
Options:
```javascript
var options = {
  auto: 'placeholders',
  fields: {
    date: {
      label: 'Datum van bezoek',
      mode: 'date'
    },
    name: {
      placeholder: 'Naam'
    },
    email: {
      keyboardType: 'default' //'email-address'
    },
    company: {
      placeholder: 'Bedrijfsnaam'
    },
    comment: {
      placeholder: 'Opmerking',
      multiline: true
    }
  }
};
```
I'm not sure if the form generator is causing this issue or that it is coming from React Native itself.
I created the project with the react-native-cli tool.

**Comment by gcanti** [comment]
> I'm not sure if the form generator is causing this issue or that it is coming from React Native itself

https://github.com/facebook/react-native/issues/279

**Comment by basdelfos** [comment]
Thanks :smirk:

**Comment by yanxi-com** [comment]
this issue is fixed by facebook/react-native#991 . https://github.com/gcanti/tcomb-form-native/pull/15 is a fix on a little issue after using the latest react-native.
